src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -4528,6 +4528,46 @@ class spell_gen_impatient_mind : public SpellScriptLoader
         }
 };
 
+class spell_gen_pvp_rules_enabled : public SpellScriptLoader
+{
+    public:
+        spell_gen_pvp_rules_enabled() : SpellScriptLoader("spell_gen_pvp_rules_enabled") { }
+
+        class spell_gen_pvp_rules_enabled_AuraScript : public AuraScript
+        {
+            PrepareAuraScript(spell_gen_pvp_rules_enabled_AuraScript);
+
+            bool Validate(SpellInfo const* /*spell*/) override
+            {
+                return ValidateSpellInfo({ SPELL_PVP_RULES_ENABLED }); // SPELL_PVP_RULES_ENABLED - defined in Player.h
+            }
+
+            void OnApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+            {
+                GetAura()->SetDuration(-1);
+                if (Player* player = GetTarget()->ToPlayer())
+                    player->TogglePvpTalents(true);
```

Comment:
I don't know but seeing how only this spell and 211128 Black Rook Rumble has it, it might control pvp talents :p